test/test_pgen2.py

```diff
@@ -87,6 +87,17 @@ def test_async_for(works_ge_py35):
     works_ge_py35.parse("async def foo():\n async for a in b: pass")
 
 
+@pytest.mark.skipif('sys.version_info[:2] < (3, 6)')
+def test_async_for_comprehension_newline():
+    # Issue #139
+    _parse("""
+async def foo():
+    [1 async 
+    for a in b 
+    ]
+""")
+
+
```

Comment:
It would be actually nicer to add something like `works_ge_py36` to `conftest.py` and then use that fixture.  Also please add some other tests with different versions such as these;
```py

async def foo():
    [1
    async
    for a in b 
    ]

async def foo():
    [1 async 
    for a in b 
    ]

async def foo():
    [
        1 async for a in b 
    ]
```
and possibly more.

Reply:
:+1: for `works_ge_py36`.

Reply:
Updated !